### PR TITLE
Adds medium sized vessel the Congregation, with chapel, soup kitchen,…

### PR DESCRIPTION
**Pull request by anonm0135**
… medbay, and dorms to give opportunities for both charity and small earnings with an RP not moneymaking focus. (PLEASE LOOK THIS OVER, FIRST TIME SHUTTLE BUILDING.)

## About the PR
I created the mid sized shuttle the LB Congregation, a chaplain centric vessel with an RP focus to enable 'charity' work or charge for the services it can provide. The Ship is centered around a modest chapel, with a chaplains office, a small 'soup kitchen', a small medbay to treat people, and 6 2x2l dorm rooms with buttons that can be networked to enable bolting of their doors. Allowing a safe place for players to SSD without breaking immersion or being looted. Other features include a TV camera in the chapel, two hydro trays for the kitchen, and 2 withdraw only ATMs for charging services or donations. RP is the focus of this vessel not moneymaking but I believe between the kitchen, med, dorms, and 'donations' the owner could stand to earn a little bit during the course of their RP.

## Why / Balance
The mission is a great small shuttle, but lacks the facilities or space to enable more serious religious rp via charitable work. This vessel will enable larger groupings to take on the religious angle in rp in a variety of ways and encourages player interaction. It also could probably quickly result in cults doing cult things, maybe.

## Technical details
Ship appraised for like 32k, I set the price to 35k for now, as this isn't quite a moneymaker I figure a small margin is better for folks relative to its price. Uses 2 plasma generators by default. 300 something tiles so it is on the smaller end of midsized to maybe avoid clogging docks.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

**Comment by Everturning** [comment]
Image?

**Comment by anonm0135** [comment]
@Everturning Sorry I had the github links in there, must have messed something up when I posted it though, here they are:

image
image
image

**Comment by Everturning** [comment]
looks sick, if not a bit more than "medium" size

**Comment by anonm0135** [comment]
Glad it looks decent, I hadn't intended it to be that size but it kinda just happened

**Comment by dvir001** [comment]
Either commit the old SpaceStation14.sln back or delete it from this commit since it was added by mistake.

**Comment by anonm0135** [comment]
> Either commit the old SpaceStation14.sln back or delete it from this commit since it was added by mistake.

I think I deleted it properly? Not 100%. Please let me know if anything else is needed on that.

**Comment by NeviNovat** [comment]
I think that all windows need to have grills under them, like in other ships, might be a standard.
Looks cool and can be upgraded or modified to serve another purpose if need be!

File: Resources/Maps/Shuttles/congregation.yml
Comment (dvir001) [inline Resources/Maps/Shuttles/congregation.yml:1]: There is a captain filled locker yet you added the same items it has again by mistake, remove the duplicated stuff and keep it as the base filled captain hardsuit locker please

Comment (anonm0135) [reply Resources/Maps/Shuttles/congregation.yml:1]: > There is a captain filled locker yet you added the same items it has again by mistake, remove the duplicated stuff and keep it as the base filled captain hardsuit locker please.
> 
> You also left a set of "ArmorCult", that's an EMAG armor set with stats of 50% protection against almost all damage types.

I was looking at a recent lilten video and it mentioned it was identical to captains armor. Do I need to remove it or could I simply not have the captains locker at all in exchange?

If the cult armor is a for sure nogo I will remove it and fix the cap locker.

Comment (dvir001) [reply Resources/Maps/Shuttles/congregation.yml:1]: Captain armor in this fork is much much lower on stats

**Comment by dvir001** [comment]
Also remove all wireless cameras routers / boards.

It exists on frontier with unlimited range. Having more then one exist has issues

**Comment by Cheackraze** [comment]
I love the idea and design behind this ship. But I noticed something when testing. It appears that this map file was initialized, and isnt usable. You may have a backup file in the auto-save folder you can go back to. When mapping, always make sure to save a copy of it before you initialize and test, or it will break the map.

**Comment by Cheackraze** [comment]
I did however update everything and fix the .sln issue for you. Be sure you are updating your project master before you continue working by using the fetch, pull, and sync options. If you need any more assistance, feel free to get ahold of us in the discord.

**Comment by Everturning** [comment]
looks sick, will use if merged

**Comment by anonm0135** [comment]
> I did however update everything and fix the .sln issue for you. Be sure you are updating your project master before you continue working by using the fetch, pull, and sync options. If you need any more assistance, feel free to get ahold of us in the discord.

I'll rebuild it from scratch and resubmit here, I have a few changes now that I have had the chance to test it a bit that I want to make. Thank you all for your help and patience thus far.

**Review by MagnusCrowe — CHANGES_REQUESTED** [review]
Please reduce your med bay to just a morgue.  I don't want this ship to step on the toes of medical specialized ships.  Instead, you might consider filling that space with a **confessional** area.

I would like to see chaplain's focus on gaining the favor of the gods to revive people who are too rotten to be cloned, storing bodies until they can be revived, and using their chaplain powers to heal.

**Comment by MagnusCrowe** [comment]
Going to close this for now because of inactivity.  Feel free to open it again when you're ready to start working on it again.

**Pull request closed without merging**